### ISSUE TITLE
[MdTooltip] `md-active` lose effectiveness after hover and mouseout

### ISSUE THREAD

**Issue by VdustR** [vuematerial/vue-material#1445]
### Steps to reproduce

hover to an active tooltip target and mouse out the target.

### Which browser?

All.

Chrome 63.0.3239.132 (64bit) and Firefox 57.0.4 (64-bit) tested.

`vue-material@1.0.0-beta-7`

### What is expected?

I think it should keep active or inactive as `md-active` set.

### What is actually happening?

It's active while mouse in and inactive while mouse out.

### Reproduction Link

Use the **Dynamically show a tooltip** example and press the button to active the tooltip, then hover and mouse out the button.

https://vuematerial.io/components/tooltip

**Comment by marcosmoura** [comment]
This is the expected behaviour. Because the user did an interaction with the element. Maybe on the future we can create an option to force the `md-active` with a strict behaviour, in a way to preserve the state!

**Comment by VdustR** [comment]
Yes, I forgot that `md-active` is a `sync` props. It should be another props to improve this.

**Comment by ujjwalabollam** [comment]
Is there a way to achieve this in Vue-material v0.8.1
